Core/Core/Assets.xcassets/InstIcons/miniArrowStartSolid.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "miniArrowStart.pdf"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "properties" : {
+    "template-rendering-intent" : "template",
+    "preserves-vector-representation" : true
```

Comment:
these probably need to mirrorX in RTL. Theres a check in the build-icons script to add that. We just need start & end to trigger that too.